README.md

```diff
@@ -1,64 +1,54 @@
 
-Opaque is a package for Apache Spark SQL that enables encryption for DataFrames using Intel SGX trusted hardware. The aim is to enable analytics on sensitive data in an untrusted cloud. Once the contents of a DataFrame are encrypted, subsequent operations will run within SGX enclaves.
+Opaque is a package for Apache Spark SQL that enables encryption for DataFrames using the OpenEnclave framework. The aim is to enable analytics on sensitive data in an untrusted cloud. Once the contents of a DataFrame are encrypted, subsequent operations will run within hardware enclaves (such as Intel SGX).
 
-This project is based on our NSDI 2017 paper [1]. The oblivious execution mode is not included in this release.
+This project is based on the following NSDI 2017 paper [1]. The oblivious execution mode is not included in this release.
 
 This is an alpha preview of Opaque, which means the software is still in development (not production-ready!). It currently has the following limitations:
 
 - Unlike the Spark cluster, the master must be run within a trusted environment (e.g., on the client).
 
 - Not all Spark SQL operations are supported. UDFs must be [implemented in C++](#user-defined-functions-udfs).
 
-- Computation integrity verification (section 4.2 of the NSDI paper) is not included.
+- Computation integrity verification (section 4.2 of the NSDI paper) is currently work in progress.
 
 [1] Wenting Zheng, Ankur Dave, Jethro Beekman, Raluca Ada Popa, Joseph Gonzalez, and Ion Stoica.
 [Opaque: An Oblivious and Encrypted Distributed Analytics Platform](https://people.eecs.berkeley.edu/~wzheng/opaque.pdf). NSDI 2017, March 2017.
 
 ## Installation
 
-After downloading the Opaque codebase, build and test it as follows. (Alternatively, we offer a [Docker image](docker/) that contains a prebuilt version of Opaque.)
+After downloading the Opaque codebase, build and test it as follows.
 
-1. Install dependencies and the [Intel SGX SDK](https://01.org/intel-software-guard-extensions/downloads):
+1. Install dependencies and the [OpenEnclave SDK](https://github.com/openenclave/openenclave/blob/v0.9.x/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md). We currently support OE version 0.9.0 (so please install with `open-enclave=0.9.0`) and Ubuntu 18.04.
 
     ```sh
-    # For Ubuntu 16.04 or 18.04:
-    sudo apt install wget build-essential openjdk-8-jdk python cmake libssl-dev
-
-    # For Ubuntu 16.04:
-    wget -O sgx_installer.bin https://download.01.org/intel-sgx/linux-2.3.1/ubuntu16.04/sgx_linux_x64_sdk_2.3.101.46683.bin
     # For Ubuntu 18.04:
-    wget -O sgx_installer.bin https://download.01.org/intel-sgx/linux-2.3.1/ubuntu18.04/sgx_linux_x64_sdk_2.3.101.46683.bin
-
-    # Installer will prompt for install path, which can be user-local
-    chmod +x ./sgx_installer.bin
-    ./sgx_installer.bin
-
-    source sgxsdk/environment
+    sudo apt install wget build-essential openjdk-8-jdk python libssl-dev
```

Comment:
Shouldn't this also include `open-enclave=0.9.0`?

Reply:
The instructions for installing OpenEnclave and its dependencies are linked in the README. These are the Opaque-specific instructions.